tcf_website/templates/course/course_professor.html

```diff
@@ -164,24 +184,33 @@ <h3 class="font-weight-light text-tcf-orange">
         </div>
 
         <div class="reviews-container mb-5 px-3">
-            <div class="reviews-toolbar d-flex">
-                <div class="ml-auto dropdown btn-group">
-                    <button class="btn bg-white" disabled>Sort by</button>
-                    <button id="review-sort-select" class="btn btn-primary dropdown-toggle" type="button" id="sortOptions"
-                            data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"></button>
-                    <div class="dropdown-menu" aria-labelledby="sortOptions">
-                        <button id="votes-sort-btn" onclick="sortReviews(this.id)" class="dropdown-item" type="button">
-                            Most Helpful
-                        </button>
-                        <button id="recent-sort-btn" onclick="sortReviews(this.id)" class="dropdown-item" type="button">
-                            Most Recent
-                        </button>
-                        <button id="highrating-sort-btn" onclick="sortReviews(this.id)" class="dropdown-item" type="button">
-                            Highest Rating
-                        </button>
-                        <button id="lowrating-sort-btn" onclick="sortReviews(this.id)" class="dropdown-item" type="button">
-                            Lowest Rating
-                        </button>
+            <div class="reviews-toolbar d-flex flex-md-row flex-column justify-content-left justify-content-md-center">
+                <div class="reviews-header d-inline-flex">
+                    <h3 class="h2">{{ reviews|length }} Comments</h3>
```

Comment:
What about Reviews instead of Comments?
```suggestion
                    <h3 class="h2">{{ reviews|length }} Reviews</h3>
```

Reply:
Sure, why not!

Reply:
Actually, the list of reviews passed from the view is actually of all reviews _with text._ Since we allow reviews to be created without comments, changing this would almost always end up claiming that we have less reviews than we actually do.

I think we should keep it as Comments, as otherwise we'd either be inconsistent with the terminology (i.e. undercounting reviews) or what's on the page (i.e. claiming some higher number than the number of comments listed, which might get confusing).